api/proto/teleport/access_graph/v1/authorized_key.proto

```diff
@@ -27,7 +27,7 @@ message AuthorizedKey {
   teleport.header.v1.Metadata metadata = 1;
   // kind is a resource kind.
   string kind = 2;
-  // sub_kind is an optional resource sub kind, used in some resources.
+  // an optional resource sub kind, used in some resources.
```

Comment:
The generated go code was not strictly following the go guidelines, but removing the symbnol name from the description will make things worse. According to the go guidelines:

> As with packages (above) and funcs (below), doc comments for types start with complete sentences naming the declared symbol. An explicit subject often makes the wording clearer, and it makes the text easier to search, whether on a web page or a command line.

I'm not sure how much we care about following go guidelines on generated go comments, but it feels wrong to have to break our style guidelines here. Maybe ask for a 3rd engineer pov before merging this.